jck/compiler.api/playlist.xml

```diff
@@ -36,6 +36,16 @@
 			<version>13</version>
 			<version>14</version>
 		</versions>
+		<disables>
+			<disable>
+				<comment>Disabled on all platforms and Java levels due to backlog/issues/559. To be run manually by CR team</comment>
+				<impl>ibm</impl>
+			</disable>
+			<disable>
+				<comment>Disabled on all platforms and Java levels due to backlog/issues/559. To be run manually by CR team</comment>
+				<impl>openj9</impl>
+			</disable>
+		</disables>
```

Comment:
Could we add `<disables>` after the `<testCaseName>`? We need to format this properly for running the disabled test pipeline in the future.

Reply:
Sure. Updated!